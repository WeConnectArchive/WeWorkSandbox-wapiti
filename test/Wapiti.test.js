import React from 'react';
import {Wapiti} from '../build/wapiti.module';
import renderer from 'react-test-renderer';

describe('Wapiti', () => {
  const component = renderer.create(
    <Wapiti />
  );

  const componentNotFound = renderer.create(
    <Wapiti 
       specUrl="/notfound.yaml"
       specType="rest"
    />
  );

  let tree;

  it('should render correctly', () => {
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should put error message', () => {
    tree = componentNotFound.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
