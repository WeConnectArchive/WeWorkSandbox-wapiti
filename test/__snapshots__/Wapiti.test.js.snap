// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wapiti should put error message 1`] = `
<div
  className="Wapiti_body-container__SUwYy FontStyles_regular-font__3G8fz"
>
  <div
    style={
      Object {
        "margin": "16px",
        "textAlign": "center",
      }
    }
  >
    Could not load the REST API spec
     
  </div>
</div>
`;

exports[`Wapiti should render correctly 1`] = `
<div
  className="Wapiti_body-container__SUwYy FontStyles_regular-font__3G8fz"
/>
`;
